ui/components/ui/popover/popover.component.js

```diff
@@ -17,6 +17,7 @@ const Popover = ({
   showArrow,
```

Comment:
Per the designs, there is no 'X' button (closure happens upon clicking "Got it!" at the bottom of the popup), and the title is centered.